gradle.properties

```diff
@@ -1,7 +1,11 @@
 version=1.2.6-SNAPSHOT
 org.gradle.caching=true
 org.gradle.parallel=true
+org.gradle.jvmargs=-Xmx2048m
 kafkaVersion=2.8.0
 avroVersion=1.10.2
-confluentVersion=6.2.0
+confluentVersion=7.1.1
```

Comment:
If you need to bump confluent, then please also bump Kafka to 3.1.0. Otherwise leave Kafka, Confluent and fluent Kafka as it was

Reply:
the bump is required for protobuf support in the schemaregistrymock